src/NuGet.Services.Staging.Manager/Filters/StageIdFilter.cs

```diff
@@ -2,36 +2,36 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using Microsoft.AspNet.Mvc;
-using Microsoft.AspNet.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
 
 namespace NuGet.Services.Staging.Manager.Filters
 {
+    public class StageIdFilterServiceAttribute : ServiceFilterAttribute

```

Comment:
Fixed

---
In reply to: [62967580](https://github.com/NuGet/NuGet.Staging/pull/44#discussion_r62967580) [](ancestors = 62967580)